### ISSUE TITLE
Move Microsoft.AspNetCore.Components.WebAssembly.Server into the shared framework

### ISSUE THREAD

**Issue by danroth27** [dotnet/aspnetcore#25643]
Now that Blazor WebAssembly ships as part of .NET 5, we should consider moving Microsoft.AspNetCore.Components.WebAssembly.Server into the shared framework.

**Comment by danroth27** [comment]
Before we can do this, we need separate the debug proxy from this package. @captainsafia

**Comment by captainsafia** [comment]
> Before we can do this, we need separate the debug proxy from this package. 

This should be doable. It'd be superseded by the work to move the DebugProxy to the SDK.

**Comment by captainsafia** [comment]
>  It'd be superseded by the work to move the DebugProxy to the SDK.

Actually, we shouldn't move the DebugProxy into the SDK.

Since we're launching the DebugProxy from VS Code (and eventually VS) we don't need to repackage it and launch it from our debugging middleware. We can just have the IDEs reference the package directly. This gives us one less thing to worry about and avoids us having to repackage things.

**Comment by mkArtakMSFT** [comment]
Closing as there doesn't seem to be benefit for customers from this work.
